src/modules/clark-reports/ClarkReportsController.ts

```diff
@@ -0,0 +1,41 @@
+import { Request, Router } from "express";
+import proxy = require("express-http-proxy");
+import { Controller } from "../../interfaces/Controller";
+import { REPORTS_ROUTES } from "../../routes";
+
+const CLARK_REPORTS_API = process.env.CLARK_REPORTS_LAMBDA_URI;
+
+export class ClarkReportsController implements Controller {
+  buildRouter(): Router {
+    const router = Router();
+
+    /**
+     * @swagger
+     * /reports:
+     * patch:
+     *    description: Get all reports
+     *    tags:
+     *      - Clark Reports
+     *    responses:
+     *        200:
+     *            description: OK
+     *        400:
```

Comment:
What would a 400 be a result of?

Reply:
It would be for Missing email or name in body